avm/res/web/serverfarm/tests/e2e/waf-aligned/main.test.bicep

```
@@ -0,0 +1,106 @@
+targetScope = 'subscription'
+
+metadata name = 'Using Well Architected parameter set'
+metadata description = 'This instance deploys the module with a WAF Aligned set of parameters.'
```

```suggestion
metadata name = 'WAF-aligned'
metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
```

In alignment with other modules